src/test/java/roomescape/controller/ReservationControllerTest.java

```diff
@@ -0,0 +1,74 @@
+package roomescape.controller;
+
+import io.restassured.RestAssured;
+import io.restassured.http.ContentType;
+import java.time.LocalDate;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.context.SpringBootTest.WebEnvironment;
+import org.springframework.boot.test.web.server.LocalServerPort;
+import org.springframework.jdbc.core.JdbcTemplate;
+import org.springframework.test.annotation.DirtiesContext;
+import roomescape.dto.ReservationRequest;
+
+@SpringBootTest(webEnvironment = WebEnvironment.RANDOM_PORT)
+@DirtiesContext(classMode = DirtiesContext.ClassMode.BEFORE_EACH_TEST_METHOD)
+class ReservationControllerTest {
+    @LocalServerPort
+    int port;
+    @Autowired
+    private JdbcTemplate jdbcTemplate;
+
+    @BeforeEach
+    void initializePort() {
+        RestAssured.port = port;
+    }
+
+    @Test
+    @DisplayName("예약 생성을 요청하면 201 CREATE를 응답한다.")
+    void postReservationTest() {
+        String sql = "insert into reservation_time (start_at) values (?)";
+        jdbcTemplate.update(sql, "11:00");
+
+        ReservationRequest reservationRequest = new ReservationRequest(
+                "브라운", LocalDate.of(2023, 8, 5), 1L
+        );
+
+        RestAssured.given().log().all()
+                .contentType(ContentType.JSON)
+                .body(reservationRequest)
+                .when().post("/reservations")
+                .then().log().all()
+                .statusCode(201);
```

Comment:
아래와 같이 `import org.springframework.http.HttpStatus` enum을 사용해주셔도 좋아요 :)
```suggestion
                .statusCode(HttpStatus.CREATED.value());
```

Reply:
해당 형태로 작성하면 훨씬 더 가독성 좋은 형태로 작성할 수 있을 것 같아 도입하였습니다! 🥰